pkg/kubernetes/kubernetes.go

```diff
@@ -206,7 +206,14 @@ func (k Kubernetes) GetKubernetesRESTConfig(masterURL, secretName, namespace str
 		logger.Error(err, "unable to find Secret", "secret name", secretName)
 		return nil, err
 	}
-
+	
+	// We want to connect the opposite site using "infinispan-operator" secret with token value inside
+	if token, ok := secret.Data["token"]; ok {
```

Comment:
Can you explain why this change is required?

Reply:
It is not related to the "Istio Feature", but anyway, this logic was taken from the previous  [infinispan-operator 1.1.x](https://github.com/infinispan/infinispan-operator/blob/1.1.x/pkg/controller/infinispan/infinispan_controller.go#L1826) version
and we are still successfully using that because working with token value (infinispan-operator's namespace scope) from the backend side is more safety than using the _whole_  admin.conf credentials (Cluster scope).
Please advise me if you have a better idea.
Thanks.

Reply:
You've added this code to the `GetKubernetesRESTConfig` which shouldn't be called when the Openshift url scheme is used for backup site Urls. The `token` value is still part of `GetOpenShiftRESTConfig` and is used with the Openshift scheme.

Reply:
Yes, you right, I have changed schema to `openshift://` and it works well with token value on Vanilla Kubernetes clusters :)